Doc/library/sys.rst

```diff
@@ -2152,11 +2152,17 @@ always available. Unless explicitly noted otherwise, all variables are read-only
 
    The default hook formats :attr:`!err_msg` and :attr:`!object` as:
    ``f'{err_msg}: {object!r}'``; use "Exception ignored in" error message
-   if :attr:`!err_msg` is ``None``.
+   if :attr:`!err_msg` is ``None``. Similar to the :mod:`traceback` module,
+   this adds color to exceptions by default. This can be disabled using
+   :ref:`environment variables <using-on-controlling-color>`.
 
    :func:`sys.unraisablehook` can be overridden to control how unraisable
    exceptions are handled.
 
+   .. versionchanged:: next
+
+      Exceptions are now printed with colorful text.
```

Comment:
```suggestion
   .. versionchanged:: next
      Exceptions are now printed with colorful text.
```

The body of versionXXX directives should be sticked to the directive name. Yes I know it's weird but that's Sphinx thing!